Chatify/chat/urls.py

```diff
@@ -1,18 +1,30 @@
 from .views import register, login, chat
 from django.urls import path
-from .api import RegistrationApi, UserListAPI
+from .api import (
+    RegistrationApi,
+    UserListAPI,
+    VisibilityStatusAPI,
+    LoginAPIView,
+    OnlineUserAPI,
+    LogoutView,
+)
 
+app_name = "chat"
 
 urlpatterns_api = [
     path("api/register/", RegistrationApi.as_view(), name="RegistrationApi"),
     path("api/user/", UserListAPI.as_view(), name="UserListAPI"),
+    path("api/visibility-status/", VisibilityStatusAPI.as_view(), name="online"),
```

Comment:
still i can see name online in 3rd parameter, I am not in favour of spoon feeding